osmaxx-py/osmaxx/excerptexport/static/excerptexport/scripts/formValidation.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+(function() {
+    var FormValidator = function() {
+        this.submitButton = document.getElementById('submit-id-submit');
+        this.extractionFormatCheckboxes = document.querySelectorAll('#div_id_formats input[type="checkbox"]');
+
+
+        this.validity = {
+            'extractionFormats': false
+        };
+
+        this.isFormValid = function() {
+            var isValid = true;
+            Object.keys(this.validity).forEach(function(key){
+                isValid = isValid && this.validity[key];
+            }.bind(this));
+            return isValid;
+        };
```

Comment:
Pity that [`arr.every(callback[, thisArg])`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) isn't available in ECMA-Script < 5. Should we use the polyfill rather than implementing this ourselves? (Our own implementation here is enormously simpler and covers our needs, I guess.)